hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/statemachine/background/BlockDeletingService.java

```diff
@@ -489,10 +488,11 @@ private ContainerBackgroundTaskResult deleteViaTransactionStore(
         Handler handler = Objects.requireNonNull(ozoneContainer.getDispatcher()
             .getHandler(container.getContainerType()));
 
-        Pair<Integer, Long> deleteBlocksResult =
+        DeleteTransactionStats deleteBlocksResult =
             deleteTransactions(delBlocks, handler, blockDataTable, container);
-        int deletedBlocksCount = deleteBlocksResult.getLeft();
-        long releasedBytes = deleteBlocksResult.getRight();
+        int deletedBlocksProceeded = deleteBlocksResult.getBlocksProcessed();
```

Comment:
Why it's called `Proceeded` here?

Reply:
If we delete Txn, all blocks in the TXN will be processed, but part of them may not be recorded in the DB and thus those shall not be counted as deleted, only processed. 
Only the blocks which successfully go through putBlock request and are recorded in DB should be counted as deleted here to ensure the block count calculation right.

Reply:
So there is a typo here, isn't it?

Reply:
Oops, spelling mistake! thx for pointing out this.